pages/hn/[chapter].js

```diff
@@ -159,7 +168,7 @@ const Chapter = ({ homepage,chapter, chapters }) => {
         </main>
       ) : (
         <div className="no-content">
-          <p>To be updated soon</p>
+          <p>यह सेक्शन जल्द ही अपडेट किया जायेगा |</p>
```

Comment:
ideally, this type of content should come from CMS in case it needs to updated later on as well